src/patternfly/components/DataList/data-list.scss

```diff
@@ -166,6 +166,10 @@
       font-size: var(--pf-c-data-list--m-compact__check--FontSize);
     }
   }
+
+  &.pf-m-drag-over {
```

Comment:
is this modifier being added to the HTML anywhere ? or is it just for react

Reply:
It's just for react to apply once the drag event starts. You can see it being applied as an inline style now in the react component. This just assigns it to a class so react can apply that instead of an inline style.